source/SIL.AppBuilder.Portal.Frontend/src/data/live.tsx

```diff
@@ -107,9 +111,10 @@ function useSubscribeToResource(connection: HubConnection<DataHub>, subscribeTo,
       }
 
       if (!isConnected) return;
-      if (connection.hubConnection.state !== ConnectionStatus.connected) return;
```

Comment:
I don't think this is ever needed. If there is no connection, there is no need to unsubscribe, and and I *think* the socket will handle that, but I don't know if I've run in to this yet. the console log doesn't print when I expect it would (never, where I expect it to print always, given the observed behavior)